UITests/Media.cs

```diff
@@ -24,11 +24,7 @@ public static void ClassInitialize(TestContext context)
             var button = session.FindElementByName("MediaPlayerElement");
             button.Click();
             var mediaElements = session.FindElementsByClassName("MediaPlayerElement");
-            Assert.IsTrue(mediaElements.Count >= 2);
-            mediaElement1 = mediaElements[0];
```

Comment:
Should the mediaElement1 and mediaElement2 variables defined on lines 15 and 16 also be deleted?

Reply:
Removed.